chart/Chart.yaml

```diff
@@ -22,3 +22,8 @@ annotations:
       links:
```

Comment:
the file should only contain current changes, as the complete history is assembled in artifacthub.io

Reply:
Okay, I hadn't realised that.